content/applications/finance/accounting/receivables/customer_payments/recording.rst

```diff
@@ -68,17 +95,14 @@ clicking on :guilabel:`ADD` under :guilabel:`Outstanding Credits` or :guilabel:`
 The invoice or bill is now marked as :guilabel:`In payment` until it is reconciled with the bank
 statement.
 
-.. seealso::
-   - :doc:`../../bank/reconciliation/use_cases`
-
 Batch payment
 -------------
 
-Batch payments allow you to group different payments to ease reconciliation. They are also useful
-when you deposit checks to the bank or for SEPA Payments.
-Go to :menuselection:`Accounting --> Customers --> Batch Payments` or :menuselection:`Accounting -->
+Batch payments allow you to group different payments to ease :doc:`reconciliation <../../bank/reconciliation/use_cases>`.
+They are also useful when you deposit checks to the bank or for SEPA Payments. To do so go to
```

Comment:
```suggestion
They are also useful when you deposit checks to the bank or for SEPA Payments. To do so, go to
```

Reply:
Ignoring these changes so far as are part of another doc